fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalSort.java

```diff
@@ -84,6 +84,24 @@ public String toString() {
         return "Sort (" + StringUtils.join(orderKeys, ", ") + ")";
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        LogicalSort that = (LogicalSort) o;
+        return offset == that.offset && Objects.equals(orderKeys, that.orderKeys);
```

Comment:
logical sort has no limit, should we remove offset?

Reply:
we should remove all limit/offset from AbstractPlan/Plan, and provide a Limit plan

Reply:
Yes, But we can do it in another PR (support limit operator)